yarn.lock

```diff
@@ -11033,6 +11050,13 @@ gather-stream@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/gather-stream/-/gather-stream-1.0.0.tgz#b33994af457a8115700d410f317733cbe7a0904b"
 
+gatsby-design-tokens@^2.0.2:
```

Comment:
Looks like our `yarn.lock` didn't contain `gatsby-design-tokens` at all previously despite it's being required by `gatsby-interface`. No idea why 🤷 

Adding it seems harmless though?